src/utils/finance/pipeline.py

```diff
@@ -0,0 +1,40 @@
+"""Merge raw state campaign finance into standardized schema"""
+
+import pandas as pd
+
+from utils.finance.data_source_registry import (
+    get_registered_sources,
+    load_state_modules,
+)
+
+ALL_STATE_SOURCES = get_registered_sources()
```

Comment:
(CRT 1) As I mentioned when we chatted, this makes me super uncomfortable, but I think that's mainly a function of my personal python coding conventions. I basically just don't like non-trivial code running on import. I also think you're right that it would be best not to require a `{state}.py` files for states that don't have any custom code. If you're curious we could talk about other ways to do this, but I think probably the best move is to just leave it as is.